### PR TITLE
chore(unity-bootstrap-theme): dataLayer example is readable

### PR DESCRIPTION
**Pull request by scott-williams-az**
### Description
Removed shorthand code and made data layer example more clear for others to implement

### Links

- codepen example of how the datalayer was incorrect https://codepen.io/stwill/pen/QWZVWVe

### Checklist

- [x] Unity project successfully builds from root `yarn install` & `yarn build`
- [x] Commits do not contain multiple scopes
- [x] Add/updated examples

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ copy = () => gulp.src(["./server/views/**/*"]).pipe(gulp.dest("./build/@asu"));
 // in the /server folder.
 // https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages#cname-errors
 cname = () => gulp.src(["./server/CNAME"]).pipe(gulp.dest("./build/"));
+cname = () => gulp.src(["./server/dataLayerExample.js"]).pipe(gulp.dest("./build/"));
```
Comment (davidornelas11) [inline gulpfile.js]: No errors are thrown here but this should probably have another name. It seems to overwrite the previous cname function

Comment (mlsamuelson) [reply gulpfile.js]: Yes that concerns me, too. Maybe name it `datalayer`. And then you'll have to include it in the series at the bottom of the file.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]

File: gulpfile.js (hunk 1 above)
Comment (mlsamuelson) [reply gulpfile.js]: Looks like that had been an interim state. Marked it resolved.

**Review by maycomayco — APPROVED** [review]
LGTM, great job Scott! 👏
That codepen should be included as a live example in our documentation!

**Pull request merged**